tools/python/convert_from_tensorflow.py

```diff
@@ -418,6 +482,15 @@ def dump_layers_to_file(self, f):
                     self.dump_dense_to_file(node, f)
                 continue
 
+            if node.op in self.batchnorm2code:
```

Comment:
what happens if there is ' a = b + c - d' in the model?

Reply:
This model will run according to normal operation and will not execute batchnorm

Reply:
yes, but we could not rely on that there's always no conflicts in self.batchnorm2code.

please use the similar logic as dense/conv2d.